apps/nextjs-website/src/lib/strapi/makeProps/makeProducts.ts

```diff
@@ -1,34 +1,63 @@
-import { products } from '@/_contents/products';
 import {
+  BaseProductWithRelationsCodec,
   StrapiProducts,
-  StrapiProduct,
-  StrapiBaseProduct,
 } from '../codecs/ProductCodec';
 import { Product } from '../../types/product';
 import { makeBannerLinkProps } from '@/lib/strapi/makeProps/makeBannerLink';
 
-export function mergeProductWithStaticContent(
-  attributes: Partial<StrapiProduct['attributes']> &
-    StrapiBaseProduct['attributes']
-): Product {
-  const staticProduct =
-    products.find((product) => product.slug === attributes.slug) || products[0];
+export function makeProductsProps(
+  strapiProducts: StrapiProducts
+): ReadonlyArray<Product> {
+  return strapiProducts.data.map(makeProductProps);
+}
+
+export function makeProductProps(product: StrapiProducts['data'][0]): Product {
   return {
-    ...staticProduct,
-    ...attributes,
-    logo: attributes.logo?.data.attributes || staticProduct.logo,
-    bannerLinks: attributes.bannerLinks?.map(makeBannerLinkProps) || [],
+    ...product.attributes,
+    description: product.attributes.description ?? '',
+    logo: product.attributes.logo?.data.attributes,
+    hasApiDataListPage:
+      product.attributes.api_data_list_page.data &&
+      product.attributes.api_data_list_page.data.attributes.apiData.data
+        .length > 0,
+    apiDataListPageUrl:
+      product.attributes.api_data_list_page.data &&
+      `/${product.attributes.slug}/api/${
+        product.attributes.api_data_list_page.data.attributes.apiData.data[0]
+          .attributes.apiRestDetail?.slug ??
+        product.attributes.api_data_list_page.data.attributes.apiData.data[0]
+          .attributes.apiSoapUrl
+      }`,
+    hasTutorialListPage: !!product.attributes.tutorial_list_page.data,
+    hasGuideListPage: !!product.attributes.guide_list_page.data,
+    hasOverviewPage: !!product.attributes.overview.data,
+    hasQuickstartGuidePage: !!product.attributes.quickstart_guide.data,
+    bannerLinks: product.attributes.bannerLinks?.map(makeBannerLinkProps) || [],
   };
 }
 
-export function makeProductsProps(
-  products: StrapiProducts,
-  staticProducts: ReadonlyArray<Product>
-): ReadonlyArray<Product> {
-  return [
-    ...staticProducts,
-    ...products.data.map(({ attributes }) => {
-      return mergeProductWithStaticContent(attributes);
-    }),
-  ];
+export function makeBaseProductWithRelationsCodec(
```

Comment:
I think there's a typo here:
```suggestion
export function makeBaseProductWithRelationsProps(
```